src/index.css

```diff
@@ -1,13 +1,339 @@
+  position: relative;
+  /*display: flex;*/
+  padding: 8px 12px;
+  border-radius: 4px;
+  background: rgba(64, 126, 201, 0.08);
+}
+
+.time {
+  padding: 5px;
+  right: 5px;
+  font-size: 12px;
+  color: rgba(96, 94, 92, 1);
+  text-align: right;
+}
+
+
+
+.date {
+  display: flex;
+  justify-content: center;
+  border-radius: 2px;
+  margin: 10px;
+}
+
+.date span {
+  padding: 5px 10px;
+  background: rgba(64, 126, 201, 0.08);
+  color: rgba(32, 31, 30, 1);
+  font-size: 12px;
+}
+
+.chatItem {
+  display: flex;
+  gap: 8px;
+  padding: 10px;
+  overflow: auto;
+}
+
+
+.chatItem:hover {
+  background-color: rgba(32, 31, 30, 0.04);
+}
+
+.chatItemSelected {
+  display: flex;
+  gap: 8px;
+  padding: 10px;
+  overflow: auto;
+  background-color: rgba(64, 126, 201, 0.08);
+}
+
+.chatItem img {
+  border-radius: 4px;
+}
+.allChats {
+  position: relative;
+  height: 100vh;
+  width: 350px;
+  border-right: 1px solid rgba(32, 31, 30, 0.12);
+}
+
+.chatSection {
+  width: 20%;
+}
+
+
+.chatItemListTitle {
+  padding: 5px;
+  font-size: 16px;
+  font-weight: bold;
+  border-bottom: 1px solid rgba(32, 31, 30, 0.12);
+  border-right: 1px solid rgba(32, 31, 30, 0.12);
+}
+
+.chatList {
+  cursor: pointer;
+  overflow-y: auto;
+  height: 100vh;
+
+}
+
+.empty {
+
+  border-right: 1px solid rgba(32, 31, 30, 0.12);
+}
+
+.avatarBig {
+  display: flex;
+  align-items: center;
+  padding: 1px;
+}
+
+.loading {
+  width: calc(100vw - 1210px);
+  height: calc(100vh - 100px);
+  padding: 2px;
+}
+
+.chatMessageText {
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+.chatTitle {
+  display: flex;
+  justify-content: space-between;
+  font-size: 15px;
+  line-height: 20px;
+  margin-bottom: 4px;
+
+}
+.chatTitle span {
+  font-weight: bold;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  overflow: hidden;
+}
+
+.time span {
+  font-weight: unset;
+}
+
+.chatMessage {
+  overflow: hidden;
+}
+
+.messenger {
+  display: flex;
+}
+
+.conversations {
+  display: flex;
+  flex-direction: column;
+  overflow-y: auto;
+
+}
+
+
+.loadingConversations {
+  display: flex;
+  width: calc(100vw - 300px);
+  height: 100vh;
+  justify-content: center;
+  font-size: 24px;
+  margin: 0 auto;
+  align-items: center;
+}
+
+@media screen and (max-width: 700px) {
+  div.messenger {
+    display: none;
+  }
+  div.warning {
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    font-size: 24px;
+    margin: 0 auto;
+    align-items: center;
+  }
+}
+
+@media screen and (max-width: 762px) {
+  div.loadingConversations {
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    font-size: 24px;
+    margin: 0 auto;
+    align-items: center;
+    text-align: center;
+  }
+}
+
+@media screen and (max-width: 634px) {
+  div.warning {
+    display: flex;
+    height: 100vh;
+    justify-content: center;
+    font-size: 24px;
+    margin: 0 auto;
+    align-items: center;
+    text-align: center;
+  }
+}
```

Comment:
I'm thinking we should not have move than 1 media query

Reply:
done, the rest is deleted